contrib/examples/actions/cloudslang-basic.yaml

```diff
@@ -0,0 +1,13 @@
+---
+description: Greet StackStorm from cloudslang ;) !
+enabled: true
+entry_point: ""
+name: cloudslang-basic
+pack: examples
+parameters:
+  flow_path:
+    default: "/opt/stackstorm/packs/examples/actions/workflows/print_text.sl"
+    type: string
+  inputs:
```

Comment:
While testing this change it occurred to me it would be a lot more natural if instead of taking a generic `inputs` runner parameter, each action would specify parameters it needs.

In this case, the action would define it needs a parameter `text` and then the runner would assemble the inputs dict from those action parameters.

So you would execute it by running something like this:

```bash
st2 run examples.cloudslang-basic text="hello world"
```

(the following would still work but it really shouldn't be used)

```bash
st2 run examples.cloudslang-basic inputs=text="hello world"
```

We can still leave a generic `inputs` parameter there, but a preferred approach should be for the action to specify the parameters it needs. This makes it a lot more natural and in-line with other runners. It also makes parameter introspection possible - consumer should know what parameters the action needs just by inspecting the action metadata, but that's not possible with the `inputs` approach. And this would also allow us to avoid the WebUI object issue since we could display a nice field for each non-object parameter :)

@meirwah @orius123 What do you guys think? I already have some changes locally if you agree that doing that would be better, I can also make that change.